components/shared_code/tests/shared/utils/test_log.py

```diff
@@ -0,0 +1,23 @@
+"""Unit tests for the logging utilities."""
+
+import logging
+import unittest
+
+from shared.utils.log import get_logger
+
+
+class LogTest(unittest.TestCase):
+    """Unit tests for the log utilities."""
+
+    def test_get_logger(self):
+        """Test getting the logger."""
+        self.assertEqual("quality_time.shared_code.tests.shared.utils.test_log", get_logger("shared_code").name)
+
+    def test_get_logger_indirectly(self):
+        """Test getting the logger via an extra function call."""
+
+        def get_logger_indirectly() -> logging.Logger:
+            """Return the logger."""
+            return get_logger("shared_code", call_stack_depth=1)
```

Comment:
Shouldn't this use a different `component_name`, so the test outcome is actually different from default behaviour?

Reply:
Good point.